### DISCUSSION TITLE
Need support for integrating IMIXS with our product

### DISCUSSION THREAD

**Discussion by Srihari-edu-tive** [imixs/imixs-workflow#792] [General]
Hi Team,

We are trying to integrate IMIXS with our education product, But we are not able understand the complete flow of this like
1) How to model, like how we can map our implementation classes to workflow etc.
2) How we can integrate this IMIXS tool with our product.

We have tried running the sample projects which in present in IMIXS Git hub, we are able to run that application, but this integration part and how to create our own models is not clear. 

Please help us with this and also let us know if there is any tutorial videos which we can refer for this.

**Comment by rsoika** [comment]
@Srihari-edu-tive - yes, of course we also offer you consulting, workshops or, on request, sponsored development for Imixs-Workflow or directly development services on your Open Source project.
Our prices for consulting and workshops by architects are € 1.200,00 per day excl. VAT.

In your case, I recommend a contingent of 1 to 2 days.

Would that be interesting for you? I would be very happy about a further cooperation! Please answer per E-Mail: info@imixs.com

===
Ralph
